### PR TITLE
work section and about me section added and styled

### PR DESCRIPTION
**Pull request by Oscar-Espinoza**
In this pull I added the work section and the about me section, all styled

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -41,6 +41,155 @@ <h2>I'm a software developer</h2>
     </div>
     <img src="Media/image.svg" alt="" class="squares">
   </section>
+  <section class="grid-container recent-work">
+    <div class="title">
+      <h1>My Recent Works</h1>
+      <hr>
```
Comment (sajid-munawar) [inline index.html:47]: - [x] Its very nice implementing your header in an h1 tag, please make sure that your background design is same as the figma design in order to stick to the figma design. and also don't forget to follow the typo(font , color) and spacing(margin, padding).
### Your design👇
recen
### figma design👇
recfigma

File: style.css [hunk 2]
```diff
@@ -114,19 +130,204 @@ body {
...
+}
+
+.projects {
+  justify-items: center;
+  background: #fff;
+}
+
+.project {
+  width: 85%;
+  height: 386px;
```
Comment (sajid-munawar) [inline style.css:183]: - [x] Please make sure that your spacing is following the template. I kindly suggest you to remove `width:85%` from here inorder to stick to the design and design accordingly. don't forget to follow the typo(color, font) as well, as it is the core requirement of your project.
### your design👇
recwork
### figma design👇
recefigma

**Review by sajid-munawar — CHANGES_REQUESTED** [review]
# Changes Required 
 ### Hi @Oscar-Espinoza  
 Nice job, you are almost there.
 
 
 
 ## (Highlights) Good Points: 
 * Good commit messages. :heavy_check_mark:
 * No linter errors. :heavy_check_mark:
 * Followed Github flow :heavy_check_mark:
 * Professional README file. :heavy_check_mark:
 * PR has a good title and summary :heavy_check_mark:

 ## (Changes Required) Aspects to improve: 
 * [ ]  Please check my inline comments

 
 ## [Optional] suggestions:
 * Nothing to mention.
 
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
 **Cheers and Happy coding! :clap::clap::clap:**
 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear
> Please, remember to tag @sajid-munawar in your question so I can receive the notification.

 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by anagudelogu — APPROVED** [review]
Hi @Oscar-Espinoza, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀  
Congratulations! 🎉 



## Highlights
✅ Responsive design. 💯 
✅ Linters are passing. 👍🏼 
✅ Good and descriptive commit messages. 🥇 

## Optional suggestions

- [ ] **[OPTIONAL]:** Sorry it wasn't pointed out earlier by a previous reviewer, but please consider formatting your README file 📜  with `markdown` syntax. You can do so by copying the [Microverse's template](https://github.com/microverseinc/readme-template) and modifying the relevant sections. (Hint: keep the hashtags `#`) 🛠️ 

- [ ] **[OPTIONAL]:** You did great following the design, however, there's a minor issue with a button 📟 Please, consider fixing it! 🛠️ 

|Template|Your design|
|---|---|
| image | image |



_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**